github/util_v4_organization_ip_allow_list.go

```diff
@@ -0,0 +1,66 @@
+package github
+
+import (
+	"context"
+
+	"github.com/shurcooL/githubv4"
+)
+
+type IpAllowListEntry struct {
+	ID             githubv4.String  `mapstructure:"id"`
+	Name           githubv4.String  `mapstructure:"name"`
+	AllowListValue githubv4.String  `mapstructure:"allow_list_value"`
+	IsActive       githubv4.Boolean `mapstructure:"is_active"`
+	CreatedAt      githubv4.String  `mapstructure:"created_at"`
+	UpdatedAt      githubv4.String  `mapstructure:"updated_at"`
+}
+
+/**
+ * Returns all IP allow list entries for an organization.
+ * This util function is used by both data_source and resource elements.
+ */
+func getOrganizationIpAllowListEntries(meta interface{}) ([]IpAllowListEntry, error) {
```

Comment:
This logic was moved from`data_source_github_organization_ip_allow_list.go` because I wanted to make use of this data in the resource.

My first attempt was having the resource call the datasource function but I felt that was too tightly coupled and a bit hacky.